src/System.Runtime.Serialization.Json/ref/System.Runtime.Serialization.Json.cs

```diff
@@ -28,11 +28,15 @@ public sealed partial class DataContractJsonSerializer : System.Runtime.Serializ
     {
         public DataContractJsonSerializer(System.Type type) { }
         public DataContractJsonSerializer(System.Type type, System.Collections.Generic.IEnumerable<System.Type> knownTypes) { }
+//CODEDOM        public DataContractJsonSerializer(System.Type type, System.Collections.Generic.IEnumerable<System.Type> knownTypes, int maxItemsInObjectGraph, bool ignoreExtensionDataObject, System.Runtime.Serialization.IDataContractSurrogate dataContractSurrogate, bool alwaysEmitTypeInformation) { }
```

Comment:
Are these CodeDom comments needed for some reason or can we remove them?

cc @huanwu, @Lxiamail, @zhenlan